Source/WebCore/html/HTMLLinkElement.cpp

```diff
@@ -167,17 +166,22 @@ void HTMLLinkElement::setDisabledState(bool disabled)
 void HTMLLinkElement::parseAttribute(const QualifiedName& name, const AtomString& value)
 {
     if (name == relAttr) {
-        m_relAttribute = LinkRelAttribute(document(), value);
+        auto parsedRel = LinkRelAttribute(document(), value);
+        auto didMutateRel = parsedRel != m_relAttribute;
+        m_relAttribute = parsedRel;
         if (m_relList)
             m_relList->associatedAttributeValueChanged(value);
-        process();
+        if (didMutateRel)
+            process();
         return;
     }
     if (name == hrefAttr) {
         process();
```

Comment:
Do we have a test showing that setting `href` to the same value does trigger a new load, interoperable across browsers?

Reply:
Oh, it looks like Firefox & Chrome don't trigger a new load in this case either so will fix.